### ISSUE TITLE
Question of evaluation difference with paper

### ISSUE THREAD

**Issue by K2Infinity** [nakamotoo/V-GPS#3]
Hi, Thanks for sharing the code of V-GPS!
I tried to reproduce the result shown in the paper, as the default setting in the code goes, the success rate is a bit different as below:
I test 5 tasks, where 2 tasks show significant differences.
put eggplant in basket: V-GPS at 67%, whereas in the paper is 84%
move near: V-GPS at 10%, whereas in the paper is 16%, and octo is 15% (in paper 12%)
Did I miss something or some change is not made in the code?
Thank you

**Comment by yifanr** [comment]
Same problem here. 

When using their checkpoint, we get similar results to the paper, by setting the eval softmax temperature to 1.

However, using their training script on the same data produces much worse results. Is there some training setting used by the authors that the codebase doesn't reflect?
